### PR TITLE
USHIFT-666: Add %{commit} and embed it into binary

### PR DESCRIPTION
**Pull request by pmtk**
ART's build system will substitute commit value with actual commit from o/microshift repo, so `microshift version -o yaml` will report it instead of printing empty string.

Also removed no longer used support for building RPM based on `github_tag `.

### CODE REVIEW

**Comment by ggiguash** [comment]
@pmtk, please update the `%changelog` section of microshift.spec.

File: packaging/rpm/microshift.spec [hunk 1]
```diff
@@ -269,9 +255,18 @@ systemctl enable --now --quiet openvswitch || true
...
+
+* Wed Nov 30 2022 Patryk Matuszak <305846+pmtk@users.noreply.github.com> 4.12.0
+- Pass %{version} to Makefile
+
 * Wed Nov 30 2022 Gregory Giguashvili <ggiguash@redhat.com> 4.12.0
 - Change the config.yaml file name to allow its overwrite by users
 
+* Mon Nov 28 2022 Patryk Matuszak <305846+pmtk@users.noreply.github.com> 4.12.0
+- Use commit time & sha for RPM and exec
+
```
Comment (ggiguash) [inline packaging/rpm/microshift.spec:269]: Can you fix your email address?

**Review by ggiguash — CHANGES_REQUESTED** [review]

File: packaging/rpm/microshift.spec (hunk 1 above)
Comment (pmtk) [reply packaging/rpm/microshift.spec:269]: You won't let me be anonymous :) Fixed

**Pull request merged**